### ISSUE TITLE
Will not plot in parallel with stagger

### ISSUE THREAD

**Issue by Crimsonking314** [abueide/harry-plotter#82]
With stagger settings set the program does not start another plot after the set stagger time. If multiple jobs are started they start all at once without delay. How is this supposed to function? Phase times do not show live either. You have to comb through the logs.

**Comment by josemanuelhda** [comment]
I agree, this is annoying for me too, now I have 2 plots writing at the same time to de final disc just because of this issue

**Comment by Crimsonking314** [comment]
Problem solved. The stagger settings only apply to the plots that you run when you click “start all”. Starting one plot at a time will not stagger them.

**Comment by josemanuelhda** [comment]
ok, thank you for the info

**Comment by CDprotector** [comment]
lol. there is no Start All button in version 1.1.12. Is this supposed to be like that or its broken ?

**Comment by abueide** [comment]
@CDprotector I renamed the start all button to start stagger to prevent confusion. If you want it to act as a start all button just set all the stagger settings to 0.

**Comment by CDprotector** [comment]
Clicking Stagger ALL will not start enything. I have 7 jobs which worked with Start ALL with 20min stagger. Now it works only by manulay starting every job.

**Comment by abueide** [comment]
@CDprotector clicking stop on a job (whether its running or not) will toggle whether its managed by the stagger manager or not. this is probably pretty confusing, but could you try clicking stop on some of your jobs after clicking start stagger and let me know if that does anything? I'll make this much more intuitive/clear for the official release

**Comment by CDprotector** [comment]
restarted pc, Start Stagger - nothing happens. CLicked Stop on individual job - nothing happens. BTW config MAX OVERALL JOBS doesnt save valua I enter there, its empty after I recheck "saved" value.

**Comment by CDprotector** [comment]
it mysteriously started after I filled config for Drives/Temps. Now it random starts even if I Stop ALL, Clear Temps and its stuck. Will wait for final version, this is little confusing.

**Comment by abueide** [comment]
@CDprotector okay thanks for testing, I'll work on this before the full release
